app/Controllers/BaseController.php

```diff
@@ -37,6 +37,13 @@ abstract class BaseController extends Controller
      */
     protected $helpers = [];
 
+     /**
+     * To declare the session propety to allow for global
+     * initialization in the initController() method
+     * See the Creation of dynamic property depreciation i php 8.2 
+     */
+    // public $session;
```

Comment:
protected modifier is fine since the code is commented

Reply:
It should be `protected`, because we don't need to set it public.